example/jupyter/tutorial_analyzer.ipynb

```diff
@@ -0,0 +1,230 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Analyzer](../../doc/analyzer_design.md) is designed to explain the machine learning model in SQLFlow. Here we are going to learn how to write an `ANALYZE` SQL.  \n",
+    "\n",
+    "We implement the analyzer based on [SHAP](https://github.com/slundberg/shap). By SQLFlow, an ANALYZE SQL will be translated to SHAP code. SQLFlow enables the code to read the dataset and load the trained model, then draws a figure to explain the model. At this stage, SQLFlow supports using the [TreeExplianer](https://github.com/slundberg/shap#tree-ensemble-example-with-treeexplainer-xgboostlightgbmcatboostscikit-learn-models) to draw a summary plot.\n",
+    "\n",
+    "## Syntax\n",
+    "\n",
+    "```SQL\n",
+    "SELECT * FROM {tbl}\n",
+    "ANALYZE {a_trained_model}\n",
+    "WITH \n",
+    "  shap_summary.{parameter}={value} ...\n",
+    "USING TreeExplainer\n",
+    "```\n",
+    "\n",
+    "- A standard SQL statement used to specify the dataset.\n",
+    "- `ANALYZE` introduces the model to explain.\n",
+    "- `USING TreeExplainer` restricts the explainer.\n",
+    "- By `WITH`, we specify the parameters to [summary_plot](https://github.com/slundberg/shap/blob/master/shap/plots/summary.py#L18-L43) with a prefix `shap_summary.`\n",
+    "  like: `shap_summary.plot_type=\"bar\"`\n",
+    "\n",
+    "\n",
+    "## The dataset\n",
+    "\n",
+    "We use the [boston housing](https://www.kaggle.com/c/boston-housing) as the demonstration dataset.      \n",
+    "First, a tree model trained to fit the dataset. Next, an `ANALYZE` SQL is used to get an overview of which features are most important for the model.\n",
```

Comment:
Please avoid passive wording so that the tutorial is more engaging. For example, "trained model" => "we train a model"